doc/sources/array_api.rst

```diff
@@ -0,0 +1,88 @@
+.. ******************************************************************************
+.. * Copyright 2024 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. _array_api:
+
+#################
+Array API support
+#################
+The `Array API <https://data-apis.org/array-api/latest/>`_ specification defines
+a standard API for all array manipulation libraries with a NumPy-like API.
+Intel(R) Extension for Scikit-Learn doesn't require
+`array-api-compat <https://github.com/data-apis/array-api-compat>`__ to be installed for
+functional support of Array API.
+In the current implementation, the functional support of array api is just part of functional
+support of different array or dataframe inputs and tries to leave presecion of the input and
+output data formats. Any array API input data provided will be converted to host numpy.ndarrays
```

Comment:
```suggestion
In the current implementation, the functional support of array api follows the functional
support of different array or DataFrame inputs and does not modify the precision of the input and
output data formats unless necessary. Any array API input data provided will be converted to host numpy.ndarrays
```

dataframe is a proper noun in this case (as most refer to a pandas/dask whatever DataFrame)